syntax = "proto2";
package Header;

message RpcHeader{
    optional string service_name = 1;
    optional string method_name = 2;
    optional int32 args_size = 3;
};